mmv1/products/networkservices/LbTrafficExtension.yaml

```diff
@@ -176,10 +174,12 @@ properties:
                 item_type: Api::Type::String
   - !ruby/object:Api::Type::Enum
     name: 'loadBalancingScheme'
+    required: true
```

Comment:
It is a breaking change to make a field required.

Can you please create a new Github issue and then change it in Terraform Provider 6.0 major release?

Reply:
Hello Zenhua, I know this is possible to be a breaking change but traffic extensions resource was merged very recently and I just wanted to reflect the same info as the [API doc](https://cloud.google.com/service-extensions/docs/reference/rest/v1beta1/projects.locations.lbTrafficExtensions#LbTrafficExtension:~:text=chains%20per%20resource.-,loadBalancingScheme,-enum%20(LoadBalancingScheme). So I dont think we need to wait to a major release to get this merged.

In case, we cannot omit this I think I better removed the required flag

Reply:
The traffic extensions resource is released to the user today and this PR will be released next week. If we make the breaking change, some users will see it. 

We still need to make the change in Provider 6.

Reply:
Hello @zli82016, I updated the PR removing the flag and I will be opening an issue for this breaking change. do you kno the release time for Provider 6?

Reply:
Thanks. The service team can make the contribution from the end of June and the the release time is the end of August this year.